### PR TITLE
UI: Add version and build info to about box

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Updates #839

### CODE REVIEW

File: client/src/services/statusService.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+class StatusService {
+  static getStatus() {
+    return axios({
+      method: 'get',
+      url: process.env.REACT_APP_STATUS_SERVICE,
+      xsrfCookieName: process.env.REACT_APP_AUTH_TOKEN,
+      xsrfHeaderName: process.env.REACT_APP_AUTH_HEADER
```
Comment (cdcabrera) [inline client/src/services/statusService.js]: So this is going sound weird, but GETs dont need the Cookie or Header with the way the server is setup, anything that modifies the data you do. And it defaults to GET, so you could trim it down it a bit
`axios({ url: ... })`

File: client/src/redux/constants/tests/index.test.js [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-import * as factsTypes from './factsConstants';
-import * as reportsTypes from './reportsConstants';
-import * as scansTypes from './scansConstants';
-import * as sourcesTypes from './sourcesConstants';
-import * as toastNotificationTypes from './toasNotificationConstants';
-import * as userTypes from './userConstants';
-import * as viewTypes from './viewConstants';
-import * as viewPaginationTypes from './viewPaginationConstants';
-import * as viewToolbarTypes from './viewToolbarConstants';
+import reduxTypes from '..';
```
Comment (cdcabrera) [inline client/src/redux/constants/tests/index.test.js]: Note: =) it's like eyes staring at me

Comment (jeff-phillips-18) [reply client/src/redux/constants/tests/index.test.js]: Redux is watching you!

Comment (cdcabrera) [reply client/src/redux/constants/tests/index.test.js]: heh

**Pull request merged**